packages/cli/src/utils/docker.ts

```diff
@@ -52,83 +25,52 @@ export const createNetwork = async (name: string, client = defaultClient) => {
   if (networks.length > 0) return networks[0]
   return client.network.create({
     CheckDuplicate: true,
-    Driver: 'overlay',
     Name: name,
-    labels: {
-      'com.docker.stack.namespace': name,
-    }
+    labels: { 'mesg.engine': 'true' }
   })
 }
 
-export const createService = async (image: string, name: string, directory: string, client = defaultClient) => {
+export const createContainer = async (image: string, name: string, directory: string, client = defaultClient) => {
   const network = await createNetwork(name, client)
-  return client.service.create({
-    Name: name,
+  const container = new Container({
```

Comment:
it will be nice to name the engine's container with `mesg_engine`

Reply:
done